{
  "repair_end": "2019-07-31 16:38:40.503416", 
  "repair_begin": "2019-07-31 16:30:12.627875", 
  "patches": [
    {
      "edits": [
        {
          "path": "src/main/java/org/apache/commons/lang3/SerializationUtils.java", 
          "line": 99, 
          "faulty": "throw new SerializationException(\"ClassNotFoundException while reading cloned object data\",ex);", 
          "type": "Replace", 
          "seed": "return object;"
        }
      ], 
      "patch": "--- /tmp/RSRepair_Defects4J_Lang_13/src/main/java/org/apache/commons/lang3/SerializationUtils.java\t2019-07-31 19:29:33.123134662 -0400\n+++ /tmp/RSRepair_Defects4J_Lang_13/patches_c9x2/Patch_66/patched/org/apache/commons/lang3/SerializationUtils.java\t2019-07-31 19:38:40.081745231 -0400\n@@ -96,7 +96,7 @@\n             return readObject;\n \n         } catch (ClassNotFoundException ex) {\n-            throw new SerializationException(\"ClassNotFoundException while reading cloned object data\", ex);\n+            return object;\n         } catch (IOException ex) {\n             throw new SerializationException(\"IOException while reading cloned object data\", ex);\n         } finally {\n\n\n"
    }
  ]
}